docs/user_authentication/third_party_token_management.md

```diff
@@ -0,0 +1,97 @@
+
+```sequence {theme: 'simple'}
+title Third-party Token Management
+
+participant Pbench-Server
+participant Browser
+participant Identity-Broker
+participant Identity-Provider
+
+autonumber 1
+activate Browser #red
+
+rbox over Browser: Dashboard
+Browser->Pbench-Server: GET Pbench client ID
+
+Pbench-Server->Browser: 200 Response 
+
+note over Pbench-Server,Browser:{Identity_Broker_auth_URI: <auth_URI>\nclient_id: <pbench_client_id>\nclient_secret: <pbench_client_secret> # optional\n}
+
+note over Browser:User clicks login
+abox over Browser: Dashboard instructs the browser to \nload identity broker authentication page
```

Comment:
Line 21 should probably somehow mention that the URL for the page to load was supplied by the Pbench Server in Step 2.  That is, the description of the step at line 14 is a little thin, and it's non-trivial for the reader to connect the dots between the contents of the note at line 18 and the contents of the abox at line 21.